eng/common/README.md

```diff
@@ -10,14 +10,3 @@ Any updates to files in the `eng/common` directory should be made in the [azure-
 All changes made will cause a PR to created in all subscribed azure-sdk language repos which will blindly replace all contents of
 the `eng/common` directory in that repo. For that reason do **NOT** make changes to files in this directory in the individual azure-sdk
 languages repos as they will be overwritten the next time an update is taken from the common azure-sdk-tools repo.
-
```

Comment:
@chidozieononiwu @sima-zhu this looks like a case were we have undone some changes that we shouldn't have undone.